src/components/Collections/Collections.js

```diff
@@ -1,19 +1,37 @@
 import React from "react";
 import axios from "axios";
+import LoadingBar from "react-top-loading-bar";
+import {
+  Container,
+  StyledDiv,
+  Title,
+  Description,
+  Total,
+  PreviewPhotos,
+  Preview,
+} from "./collections.styles";
+import unavilableCover from "../../assets/UnavailableCover.png";
 
 class Collections extends React.Component {
   state = {
     collections: null,
+    isLoading: false,
   };
 
-  baseUrl = `${process.env.REACT_APP_API_BASE_URL}/users`
+  loadingBar = React.createRef();
+
+  baseUrl = `${process.env.REACT_APP_API_BASE_URL}/users`;
 
   retrieveCollections = async (username) => {
     try {
+      this.loadingBar.current.continuousStart();
+      this.setState({ isLoading: true });
       const { data } = await axios(
         `${this.baseUrl}/${username}/collections?client_id=${process.env.REACT_APP_API_KEY}`
       );
-      this.setState({ collections: data });
+      this.setState({ collections: data, isLoading: false });
+      this.loadingBar.current.complete();
+      console.log(this.state.collections)
```

Comment:
😅

Reply:
I swear I searched for them and removed every single one...I. swear. 🤣